decimal_test.go

```diff
@@ -3697,7 +3866,7 @@ func TestDecimal_Min(t *testing.T) {
 	}
 }
 
-//nolint:predeclared
+//nolint:revive
```

Comment:
You are disabling a whole linter here. You should add a comment.

You should consider adding nolintlint linter and add `require-specific` and `require-explanation`

https://golangci-lint.run/usage/linters/#nolintlint

Also, does this directive have to be applied to the whole function? Can't it be move somewhat only where it's needed?